config/api.yml

```diff
@@ -2221,6 +2221,7 @@
     - :collection
     :verbs: *gpd
     :klass: Vm
+    :identifying_attrs: ems_ref
```

Comment:
ems_ref is not identifying by itself.  At a minimum you need ems_id/ems_ref pair.  (Think if you have 2 vcenters, it's highly likely that `vm-1` will appear in both)